plugin/plugin.go

```diff
@@ -23,6 +26,26 @@ func New(policy string) validator.Plugin {
 
 func (p *plugin) Validate(ctx context.Context, req *validator.Request) error {
 
+	log.SetFormatter(&log.JSONFormatter{})
+	log.SetOutput(os.Stdout)
```

Comment:
These modify a global object and don't need to be called on every request. You can probably just drop this in main.

Reply:
The [handler](https://github.com/drone/drone-go/blob/e68a046aa152166bbb417c7a25571c706d4115fd/plugin/validator/handler.go#L87) doesn't seem to pass loggers to plugin calls, maybe I can set those in the constructor instead?

Reply:
It doesn't need to be passed from the handler. I was just pointing out that by default logrus operates on a [global logger](https://github.com/sirupsen/logrus/blob/b50299cfaaa1bca85be76c8984070e846c7abfd2/exported.go#L11). So you only ever need to calls these once, which typically is in `main()` or `init()`.